Tests/XCUITests/LoginsTests.swift

```diff
@@ -51,6 +64,7 @@ class SaveLoginTest: BaseTestCase {
         passcodeInput.typeText("foo\n")
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2288345
```

Comment:
Should we set the "Automated Test Name(s)" on TestRail to `Tests/XCUITests/LoginTest#testLoginsListFromBrowserTabMenu` instead?